martus-client/source/org/martus/client/swingui/jfx/generic/ImportVariousTypesAction.java

```diff
@@ -120,15 +122,27 @@ protected void importBulletinFromMbaFile(File fileToImport)
 		try
 		{
 			BulletinFolder folderToImportInto = getImportFolder();
+			removeFolderContentChangedHandler(folderToImportInto);
 			getApp().getStore().importZipFileBulletin(fileToImport, folderToImportInto, false);
+			restoreFolderContentChangedHandler(folderToImportInto);
 			updateCases();
 		}
 		catch (Exception e)
 		{
 			MartusLogger.logException(e);
 		}
 	}
-	
+
```

Comment:
- The rFCCL() and rFCCH() methods should be inside the FCMController class.  would be like getFxCaseManagementController().removeFolderContentsListener(folder) and same for add
- restoreXxx should be renamed to addXxx.